src/main/java/gregtech/common/metatileentities/MetaTileEntities.java

```diff
@@ -656,6 +658,8 @@ public static void init() {
         STEAM_IMPORT_BUS = GregTechAPI.registerMetaTileEntity(1632, new MetaTileEntitySteamItemBus(gregtechId("steam_import_bus"), false));
         STEAM_HATCH = GregTechAPI.registerMetaTileEntity(1633, new MetaTileEntitySteamHatch(gregtechId("steam_hatch")));
 
+        CLIPBOARD_TILE = GregTechAPI.registerMetaTileEntity(1640, new MetaTileEntityClipboard(gregtechId("clipboard")));
```

Comment:
This ID should be compacted down to 1636. This way we have room for steam fluid hatches should we want them.